### PR TITLE
Init

### PR DESCRIPTION
**Pull request by sudheep2**

### CODE REVIEW

File: src/main/java/com/flipkart/logisticsadaptor/guice/ServerModule.java [hunk 1]
```diff
@@ -37,8 +39,28 @@ public LogisticSearchOrchestrator provideLogisticSearchOrchestrator(EkartAdaptor
...
+    }
+
+    @Provides
+    @Singleton
+    @Inject
+    @Named("EKartSearchClient")
+    public EkartAdaptorEngine provideEkartAdaptorEngineSearch( @Named("EKartSearchClient") BaseClient<SearchRequest, OnSearchMessage> searchRequestResponseMessageBaseClient){
+        EkartAdaptorEngine obj=new EkartAdaptorEngine();
+        obj.EkartAdaptorEngineSearch(searchRequestResponseMessageBaseClient);
+        return obj;
```
Comment (sakamotojin) [inline src/main/java/com/flipkart/logisticsadaptor/guice/ServerModule.java:53]: DOn't Make A new Instance Of Ekart Engine , Its Singeston

File: src/main/java/com/flipkart/logisticsadaptor/guice/ServerModule.java [hunk 2]
```diff
@@ -37,8 +39,28 @@ public LogisticSearchOrchestrator provideLogisticSearchOrchestrator(EkartAdaptor
...
+    }
+
+    @Provides
+    @Singleton
+    @Inject
+    @Named("EKartInitClient")
+    public EkartAdaptorEngine provideEkartAdaptorEngineInit(@Named("EKartInitClient")BaseClient<InitRequest, OnInitMessage> initRequestOnInitMessageBaseClient){
+        EkartAdaptorEngine obj=new EkartAdaptorEngine();
+        obj.EkartAdaptorEngineInit(initRequestOnInitMessageBaseClient);
```
Comment (sakamotojin) [inline src/main/java/com/flipkart/logisticsadaptor/guice/ServerModule.java:62]: Same here

File: src/main/java/com/flipkart/logisticsadaptor/models/Geocode.java [hunk 3]
```diff
@@ -6,6 +6,22 @@
 @Builder
 @Getter
 public class Geocode {
+    public String getLatitude() {
+        return latitude;
+    }
+
+    public void setLatitude(String latitude) {
+        this.latitude = latitude;
+    }
```
Comment (sakamotojin) [inline src/main/java/com/flipkart/logisticsadaptor/models/Geocode.java:15]: use getter setter annotations

File: config.yml [hunk 4]
```diff
@@ -10,7 +10,7 @@ server:
 database:
   driverClass: com.mysql.cj.jdbc.Driver
   user: root
-  password: root
```
Comment (sakamotojin) [inline config.yml:13]: why is this removed

File: src/main/java/com/flipkart/logisticsadaptor/guice/ServerModule.java (hunk 1 above)
Comment (sudheep2) [reply src/main/java/com/flipkart/logisticsadaptor/guice/ServerModule.java:53]: if i am keeping a single instance then i am getting a error stating programs have same arguments

File: config.yml (hunk 4 above)
Comment (sudheep2) [reply config.yml:13]: this was for my local db it doesnt have password so i removed it from my file

**Review by sakamotojin — APPROVED** [review]
